app/services/user.js

```diff
@@ -23,4 +23,28 @@ module.exports.authenticateUser = function( user, userPassword ) {
   return {
     success: true, message: 'Token is ready', token: token
   };
+};
+
+/**
+ * Verifies given token
+ *
+ * @param {JSON web token} token - The token which should be verified
+ * @returns {Object}
+ */
+module.exports.verifyToken = function( token, res, next ) {
```

Comment:
Everything is related to the User model, but if we separate our functionality in small pieces of code in different files by a context - we will improve readability and we won't violate [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle).